src/store/modules/account.js

```diff
@@ -53,6 +57,10 @@ const mutations = {
 
   addOrder (state, order) {
     state.orders.push(order)
+  },
+
+  resetCustomer () {
```

Comment:
ainda falta declarar o argumento da função pra pegar o state do contexto

```suggestion
  resetCustomer (state) {
```